timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md

```diff
@@ -0,0 +1,393 @@
+how large the time buckets are. Above are examples that create 1-day time
+buckets.
+
+## Generate insights with SQL
+In this section, you see a list of questions related to blockchain
+transactions, blocks, and miner revenue. Each question is accompanied by a
+relevant SQL query and a chart that answers the question with a brief
+explanation.
+
+**Questions**
+* Is there any connection between the number of transactions and the transaction fees?
+* Does the transaction volume affect the BTC-USD rate?
+* Do more transactions in a block mean the block is more expensive to mine?
+* What percentage of the average miner's revenue comes from fees vs. block rewards?
+* How does block weight affect miner fees?
+* What’s the average miner revenue per block?
+
+
+### Is there any connection between the number of transactions and the transaction fees?
+When it comes to blockchains, a major concern for users is the transaction
+fees. At the end of the day, if a blockchain is too expensive to use, not many
+people are willing to use it. Let’s see if there’s any correlation between the
+number of Bitcoin transactions and the fees. The time range for this analysis
+is the last day.
+```sql
+SELECT
+ bucket AS "time",
+ tx_count as "tx volume",
+ average(stats_fee_sat) as fees
+FROM one_hour_transactions
+WHERE bucket > NOW() - INTERVAL '1 day'
+ORDER BY 1
+```
+![transaction volume and fees](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/tx_volume_fees.png)
+
+On this chart, the green line indicates the average transaction volume over
+time, and the yellow line indicates the average fee per transaction over
+time. This type of insight can help you, for example, decide whether you
+should submit a transaction now or wait a couple of days for fees to
+decrease. Now let’s look at an analysis that traders might find insightful.
+
+### Does the transaction volume affect the BTC-USD rate?
+In cryptocurrency trading, there’s a lot of speculation. Finding correlations
+between fundamental blockchain metrics, like transaction volume or fees and the
+BTC-USD rate can help traders shape their strategy and make decisions based
+on data.
+
+```sql
+SELECT
+ bucket AS "time",
+ tx_count as "tx volume",
+ total_fee_usd / (total_fee_sat*0.00000001) AS "btc-usd rate"
+FROM one_hour_transactions
+WHERE bucket > NOW() - INTERVAL '1 day'
+ORDER BY 1
+```
+![volume and BTC-USD](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/volume_btc_usd.png)
+
+Again, the green line shows the average transaction volume over time and the
+yellow line shows the BTC-USD conversion rate at any given time.
+
+Next, let's segue into block-level insights by analyzing the connection
+between transactions and blocks.
+
+### Do more transactions in a block mean the block is more expensive to mine?
+For someone who is not an expert on how the Bitcoin blockchain works, it might be a
+difficult question to answer. So let’s ask the data and see if it’s true.
```

Comment:
Why is this necessary? Is this something that is often claimed, but that is not intuitive to see? If so, I would just say that. Something like:

`It's often said that a block with more transactions is more expensive to mine. Analyze the data to see if it supports this trend.`